### PR TITLE
Add r-wmtsa.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

File: recipes/r-wmtsa/meta.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ build:
   rpaths:
     - lib/R/lib/
     - lib/
+  skip: True  # [win]
```
Comment (bgruening) [inline recipes/r-wmtsa/meta.yaml]: Is win32 enough to skip?

Comment (cbrueffer) [reply recipes/r-wmtsa/meta.yaml]: No, as far as I remember the build failed on win64 too.

Comment (bgruening) [reply recipes/r-wmtsa/meta.yaml]: Would be nice if you could add a comment with he win32 error, so people interested on win32 builds could fix it.

Comment (cbrueffer) [reply recipes/r-wmtsa/meta.yaml]: win64 you mean? I'll see what the error was and add a comment.

Comment (cbrueffer) [reply recipes/r-wmtsa/meta.yaml]: Could you commit this one as is? I'll need to re-run the Travis build to check why win64 fails, I think that's easier from within a feedstock.

**Pull request merged**